packages/plugin-rename-all-fields/src/index.ts

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import type {
+  CollectionCustomizer,
+  DataSourceCustomizer,
+  Plugin,
+} from '@forestadmin/datasource-customizer';
+
+import { Options } from './types';
+
+export { Options as RenameAllFieldOption };
+
+export default function renameAllField(
+  dataSource: DataSourceCustomizer<any>,
+  collection: CollectionCustomizer<any>,
+  options?: Options,
```

Comment:
Bad naming IMO maybe just call it `handler` ?

Reply:
or `transformer` 🤖

Reply:
You know that external users don't really see it when using the `.use(renameAllFields, ..)` method ?